any hints on errors during collecting the tests?

While updating debian package for pandas 0.24.1 running into this odd error with pytest during collection.  Just wondered if someone might have an immediate clue so I don't waste too much time trying to catch the ghost (pytest pkg is 3.10.1-2):

```
collected 47434 items / 2 errors / 778 deselected / 3 skipped                                                     

===================================================== ERRORS ======================================================
_____________________________ ERROR collecting tests/indexes/datetimes/test_tools.py ______________________________
/usr/lib/python2.7/dist-packages/pluggy/hooks.py:284: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:67: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:61: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
/usr/lib/python2.7/dist-packages/_pytest/python.py:224: in pytest_pycollect_makeitem
    res = list(collector._genfunctions(name, obj))
/usr/lib/python2.7/dist-packages/_pytest/python.py:409: in _genfunctions
    self.ihook.pytest_generate_tests(metafunc=metafunc)
/usr/lib/python2.7/dist-packages/pluggy/hooks.py:284: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:67: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:61: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
/usr/lib/python2.7/dist-packages/_pytest/python.py:133: in pytest_generate_tests
    metafunc.parametrize(*marker.args, **marker.kwargs)
/usr/lib/python2.7/dist-packages/_pytest/python.py:973: in parametrize
    param_index,
/usr/lib/python2.7/dist-packages/_pytest/python.py:841: in setmulti2
    self._checkargnotcontained(arg)
/usr/lib/python2.7/dist-packages/_pytest/python.py:825: in _checkargnotcontained
    raise ValueError("duplicate %r" % (arg,))
E   ValueError: duplicate 'box'
__________________________________ ERROR collecting tests/resample/test_base.py ___________________________________
In test_resample_empty_dataframe_all_ts: function uses no argument '_index_factory'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================== 3 skipped, 778 deselected, 2 error in 37.76 seconds ===============================
```

<details>
<summary>All of it with versions etc (click to expand)</summary>

```
echo "backend : Agg" >| /build/pandas-0.24.1/build/matplotlibrc
: # Run unittests here against installed pandas
echo "2.7" | grep -q '^3' && PY=3 || PY=2.7; \
export PYTHONPATH=`/bin/ls -d $PWD/debian/tmp/usr/lib/python$PY/*/`; \
export MPLCONFIGDIR=/build/pandas-0.24.1/build HOME=/build/pandas-0.24.1/build; \
cd build/; \
python2.7 -c 'import pandas as pd; pd.show_versions()'; \
        LOCALE_OVERRIDE=C xvfb-run -a -s "-screen 0 1280x1024x24 -noreset" \
        python2.7 -m pytest -s -v -m "not single and not network and not disabled "    $PYTHONPATH/pandas;

INSTALLED VERSIONS
------------------
commit: None
python: 2.7.15.final.0
python-bits: 64
OS: Linux
OS-release: 4.18.0-0.bpo.1-amd64
machine: x86_64
processor: 
byteorder: little
LC_ALL: C
LANG: C
LOCALE: None.None

pandas: 0.24.1
pytest: 3.10.1
pip: None
setuptools: 40.7.1
Cython: 0.29.2
numpy: 1.16.1
scipy: 1.1.0
pyarrow: None
xarray: None
IPython: 5.8.0
sphinx: 1.8.3
patsy: None
dateutil: 2.7.3
pytz: 2018.9
blosc: None
bottleneck: None
tables: 3.4.4
numexpr: 2.6.9
feather: None
matplotlib: 2.2.3
openpyxl: 2.4.9
xlrd: 1.1.0
xlwt: 1.3.0
xlsxwriter: 1.1.2
lxml.etree: 4.3.0
bs4: 4.7.1
html5lib: 1.0.1
sqlalchemy: None
pymysql: None
psycopg2: None
jinja2: 2.10
s3fs: None
fastparquet: None
pandas_gbq: None
pandas_datareader: None
gcsfs: None
=============================================== test session starts ===============================================
platform linux2 -- Python 2.7.15+, pytest-3.10.1, py-1.7.0, pluggy-0.8.0 -- /usr/bin/python2.7
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/pandas-0.24.1/build/.hypothesis/examples')
rootdir: /tmp/buildd/pandas-0.24.1, inifile: setup.cfg
plugins: hypothesis-3.71.11
collected 47434 items / 2 errors / 778 deselected / 3 skipped                                                     

===================================================== ERRORS ======================================================
_____________________________ ERROR collecting tests/indexes/datetimes/test_tools.py ______________________________
/usr/lib/python2.7/dist-packages/pluggy/hooks.py:284: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:67: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:61: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
/usr/lib/python2.7/dist-packages/_pytest/python.py:224: in pytest_pycollect_makeitem
    res = list(collector._genfunctions(name, obj))
/usr/lib/python2.7/dist-packages/_pytest/python.py:409: in _genfunctions
    self.ihook.pytest_generate_tests(metafunc=metafunc)
/usr/lib/python2.7/dist-packages/pluggy/hooks.py:284: in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:67: in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
/usr/lib/python2.7/dist-packages/pluggy/manager.py:61: in <lambda>
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
/usr/lib/python2.7/dist-packages/_pytest/python.py:133: in pytest_generate_tests
    metafunc.parametrize(*marker.args, **marker.kwargs)
/usr/lib/python2.7/dist-packages/_pytest/python.py:973: in parametrize
    param_index,
/usr/lib/python2.7/dist-packages/_pytest/python.py:841: in setmulti2
    self._checkargnotcontained(arg)
/usr/lib/python2.7/dist-packages/_pytest/python.py:825: in _checkargnotcontained
    raise ValueError("duplicate %r" % (arg,))
E   ValueError: duplicate 'box'
__________________________________ ERROR collecting tests/resample/test_base.py ___________________________________
In test_resample_empty_dataframe_all_ts: function uses no argument '_index_factory'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=============================== 3 skipped, 778 deselected, 2 error in 37.76 seconds ===============================
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":99"
      after 11 requests (8 known processed) with 0 events remaining.
make[1]: *** [debian/rules:128: python-test2.7] Error 2
make[1]: Leaving directory '/build/pandas-0.24.1'
make: *** [debian/rules:67: binary] Error 2
```
</details>

--- a/pandas/tests/resample/test_base.py
+++ b/pandas/tests/resample/test_base.py
@@ -28,15 +28,10 @@ PERIOD_RANGE = (
     period_range, 'pi', datetime(2005, 1, 1), datetime(2005, 1, 10))
 TIMEDELTA_RANGE = (timedelta_range, 'tdi', '1 day', '10 day')
 
-ALL_TIMESERIES_INDEXES = [DATE_RANGE, PERIOD_RANGE, TIMEDELTA_RANGE]
-
-
-def pytest_generate_tests(metafunc):
-    # called once per each test function
-    if metafunc.function.__name__.endswith('_all_ts'):
-        metafunc.parametrize(
-            '_index_factory,_series_name,_index_start,_index_end',
-            ALL_TIMESERIES_INDEXES)
+all_ts = pytest.mark.parametrize(
+    '_index_factory,_series_name,_index_start,_index_end',
+    [DATE_RANGE, PERIOD_RANGE, TIMEDELTA_RANGE]
+)
 
 
 @pytest.fixture
@@ -84,7 +79,8 @@ def test_asfreq_fill_value(series, create_index):
     assert_frame_equal(result, expected)
 
 
-def test_resample_interpolate_all_ts(frame):
+@all_ts
+def test_resample_interpolate(frame):
     # # 12925
     df = frame
     assert_frame_equal(
@@ -101,8 +97,9 @@ def test_raises_on_non_datetimelike_index():
         xp.resample('A').mean()
 
 
+@all_ts
 @pytest.mark.parametrize('freq', ['M', 'D', 'H'])
-def test_resample_empty_series_all_ts(freq, empty_series, resample_method):
+def test_resample_empty_series(freq, empty_series, resample_method):
     # GH12771 & GH12868
 
     if resample_method == 'ohlc':
@@ -121,8 +118,9 @@ def test_resample_empty_series_all_ts(freq, empty_series, resample_method):
     assert_series_equal(result, expected, check_dtype=False)
 
 
+@all_ts
 @pytest.mark.parametrize('freq', ['M', 'D', 'H'])
-def test_resample_empty_dataframe_all_ts(empty_frame, freq, resample_method):
+def test_resample_empty_dataframe(empty_frame, freq, resample_method):
     # GH13212
     df = empty_frame
     # count retains dimensions too
@@ -162,7 +160,8 @@ def test_resample_empty_dtypes(index, dtype, resample_method):
         pass
 
 
-def test_resample_loffset_arg_type_all_ts(frame, create_index):
+@all_ts
+def test_resample_loffset_arg_type(frame, create_index):
     # GH 13218, 15002
     df = frame
     expected_means = [df.values[i:i + 2].mean()
@@ -202,7 +201,8 @@ def test_resample_loffset_arg_type_all_ts(frame, create_index):
             assert_frame_equal(result_how, expected)
 
 
-def test_apply_to_empty_series_all_ts(empty_series):
+@all_ts
+def test_apply_to_empty_series(empty_series):
     # GH 14313
     s = empty_series
     for freq in ['M', 'D', 'H']:
@@ -212,7 +212,8 @@ def test_apply_to_empty_series_all_ts(empty_series):
         assert_series_equal(result, expected, check_dtype=False)
 
 
-def test_resampler_is_iterable_all_ts(series):
+@all_ts
+def test_resampler_is_iterable(series):
     # GH 15314
     freq = 'H'
     tg = TimeGrouper(freq, convention='start')
@@ -223,7 +224,8 @@ def test_resampler_is_iterable_all_ts(series):
         assert_series_equal(rv, gv)
 
 
-def test_resample_quantile_all_ts(series):
+@all_ts
+def test_resample_quantile(series):
     # GH 15023
     s = series
     q = 0.75

